src/tokens/TabWallet.tsx

```diff
@@ -1,231 +1,83 @@
-        ['transparent', Colors.gray1]
-      ),
-    }
-  }, [scrollPosition.value, nonStickyHeaderHeight])
-
-  const animatedFooterStyles = useAnimatedStyle(() => {
-    return {
-      transform: [
-        {
-          translateY: footerPosition.value,
-        },
-      ],
-    }
-  }, [footerPosition.value])
-
-  useScrollAwareHeader({
-    navigation,
-    title: t('bottomTabsNavigator.wallet.title'),
-    scrollPosition,
-    startFadeInPosition: nonStickyHeaderHeight - HEADER_OPACITY_ANIMATION_START_OFFSET,
-    animationDistance: HEADER_OPACITY_ANIMATION_DISTANCE,
-  })
-
-  const handleMeasureNonStickyHeaderHeight = (event: LayoutChangeEvent) => {
-    setNonStickyHeaderHeight(event.nativeEvent.layout.height)
-  }
-
-  const handleMeasureListHeaderHeight = (event: LayoutChangeEvent) => {
-    setListHeaderHeight(event.nativeEvent.layout.height)
-  }
-
-  const handleMeasureListFooterHeight = (event: LayoutChangeEvent) => {
-    setListFooterHeight(event.nativeEvent.layout.height)
-  }
-
-  const handleChangeActiveView = (selectedTab: AssetTabType) => {
-    navigation.setParams({ activeAssetTab: selectedTab })
-  }
-
+  const totalTokenBalanceLocal = useTotalTokenBalance()
+  const balanceDisplay = hideWalletBalances
+    ? `XX${decimalSeparator}XX`
+    : totalTokenBalanceLocal?.toFormat(2)
   return (
-    // Transparency issue on Android present when a fragment is used - Nested Animated.View prevents it
-    <Animated.View>
-      <Animated.View
-        style={[styles.listHeaderContainer, animatedListHeaderStyles]}
-        onLayout={handleMeasureListHeaderHeight}
-      >
-        <View
-          style={[styles.nonStickyHeaderContainer]}
-          onLayout={handleMeasureNonStickyHeaderHeight}
-        >
-          <View style={styles.balanceSection}>
-            <AssetsTokenBalance showInfo={false} />
-          </View>
-          <ActionsCarousel key={'ActionsCarousel'} />
-        </View>
-        <AssetTabBar
-          activeTab={activeTab}
-          onChange={handleChangeActiveView}
-          displayPositions={displayPositions}
-        />
-      </Animated.View>
-
-      <AssetList
-        activeTab={activeTab}
-        listHeaderHeight={listHeaderHeight}
-        handleScroll={handleScroll}
-      />
-      {showClaimRewards && (
-        <Animated.View
-          style={[styles.footerContainer, animatedFooterStyles]}
-          onLayout={handleMeasureListFooterHeight}
-        >
-          <Button
-            type={BtnTypes.SECONDARY}
-            size={BtnSizes.FULL}
-            text={t('assets.claimRewards')}
+    <SafeAreaView style={styles.container} edges={['top']}>
+      <View style={styles.row}>
+        <Text style={styles.totalBalance} testID={'TotalTokenBalance'}>
+          {!hideWalletBalances && localCurrencySymbol}
+          {balanceDisplay}
+        </Text>
+        <HideBalanceButton hideBalance={hideWalletBalances} />
+      </View>
+      <ScrollView>
```

Comment:
Tried to use just `View` here, but when I did that the `TokenBalanceItem`'s were not rendering correctly at all (only showing the token icon, no text or balance).